src/commonMain/kotlin/io/github/kevincianfarini/alchemist/type/Mass.kt

```diff
@@ -84,6 +85,17 @@ public value class Mass internal constructor(internal val rawMicrograms: Saturat
      */
     public operator fun times(scale: Long): Mass = Mass(rawMicrograms * scale)
 
+    /**
+     * Returns a mass whose value is multiplied by the specified [scale].
+     *
+     * @throws IllegalArgumentException when this mass is [infinite][isInfinite] and [scale] is 0.
+     */
```

Comment:
For every quantity we need to document that this operation may lose precision. Eg. 

> Returns a mass whose value is multiplied by the specified [scale]. This operation may lose precision when the result cannot be precisely represented with a [Double] number. 
> 
> @throws IllegalArgumentException when this mass is [infinite][isInfinite] and scale is 0.0 or when this mass is 0 and scale is [infinite][Double.isInfinite].